### PR TITLE
Abhinav Gupta

### PR DESCRIPTION
**Pull request by AbhinavGupta-de**
solves #1

### CODE REVIEW

File: members/abhinavgupta-de.md [hunk 1]
```diff
@@ -0,0 +1 @@
+name: Abhinav Gupta github_handle: abhinavgupta-de
```
Comment (Diwakar-Gupta) [inline members/abhinavgupta-de.md:1]: consider changing this to
```md
name: Abhinav Gupta
github_handle: abhinavgupta-de
```

**Review by Diwakar-Gupta — CHANGES_REQUESTED** [review]

File: members/abhinavgupta-de.md (hunk 1 above)
Comment (AbhinavGupta-de) [reply members/abhinavgupta-de.md:1]: On it

**Comment by AbhinavGupta-de** [comment]
Finished the requested changes

image

**Review by Diwakar-Gupta — APPROVED** [review]
looks good

**Pull request merged**